components/brave_new_tab_ui/api/initialData.ts

```diff
@@ -63,6 +62,23 @@ export async function getInitialData (): Promise<InitialData> {
   }
 }
 
+export async function getOnlyAnonWallet (): Promise<any> {
```

Comment:
As this is now shared data among Rewards and Binance, this should be its own separate promise